clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -59,24 +59,24 @@
     public ConsumerRecords<K, V> poll(long timeout);
 
     /**
-     * @see KafkaConsumer#commit(CommitType)
+     * @see KafkaConsumer#commit()
      */
-    public void commit(CommitType commitType);
+    public void commit();
 
     /**
-     * @see KafkaConsumer#commit(CommitType, ConsumerCommitCallback)
+     * @see KafkaConsumer#commit(OffsetCommitCallback)
```

Comment:
The intent with that naming was to differentiate between cases where the user callback is invoked once upon completion of an event (commit callbacks should only be invoked once when the action completes) and cases where it may be invoked arbitrarily many times and is not tied to a specific operation (rebalance is triggered externally, so the listener methods may be invoked many times).

However, if that didn't come across, then maybe we haven't done a good enough job naming them...